app/controllers/reviews_controller.rb

```diff
@@ -44,7 +44,7 @@ def destroy
     @tattoo = @review.tattoo
     if !signed_in?
       authenticate_user!
-    elsif signed_in? && current_user == @review.user
+    elsif signed_in? && (current_user==@review.user||current_user.admin?)
```

Comment:
Surrounding space missing for operator '||'.<br>Surrounding space missing for operator '=='.